gnat2goto/driver/tree_walk.adb

```diff
@@ -1480,7 +1481,14 @@ package body Tree_Walk is
       Typecast_Expr : constant Irep :=
         Make_Op_Typecast (Value, Sloc (N), Type_Of_Val);
    begin
-      return Make_Range_Assert_Expr (N, Typecast_Expr, Type_Of_Val);
+      --  TODO: Range expressions for non-bounded types are outside
+      --        the scope of this PR
```

Comment:
fair, but remove this comment before merging, nobody can see what PR you're talking about just by looking at the code